common/changes/@itwin/manage-versions-react/sourabh-context-menu-update-for-manage-versions_2024-03-28-13-32.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/manage-versions-react",
+      "comment": "Adding Download action on Named version table to download the associated files of a Named Version from the UI. Hence, Added changeset checkpoint API to download file. Also, added context menu to provide quick access to multiple actions.",
```

Comment:
the only question would be if we dont need a prop to enable this..
but maybe its fine to try go without it :/

Reply:
@veekeys It will be Download option addition alone not much change. So, we can go without prop also. Any how last decision will be yours.